flow-report/src/summary/summary.tsx

```diff
@@ -84,7 +89,10 @@ export const SummaryFlowStep: FunctionComponent<{
         style={{width: THUMBNAIL_WIDTH, maxHeight: thumbnailHeight}}
       />
       <FlowSegment mode={lhr.gatherMode}/>
-      <a className="SummaryFlowStep__label" href={`#index=${hashIndex}`}>{label}</a>
+      <div className="SummaryFlowStep__label">
+        <div className="SummaryFlowStep__label--mode">{MODE_DESCRIPTIONS[lhr.gatherMode]}</div>
```

Comment:
nit: I would consider this more like `SummaryFlowStepLabel__mode` rather than a `--mode` modifier on `SummaryFlowStep__label` (after all, it's a child of `SummaryFlowStep__label`)